conf/config.properties.example

```diff
@@ -565,6 +565,10 @@ client.projectEditor.isRestricted.default=false
 
 client.roles.crossProjectLoaderRole=ROLE_ACAS-CROSS-PROJECT-LOADER
 
+# List of project codes to filter from ACAS
```

Comment:
"filter from ACAS" is a little unclear on whether it hides these projects in ACAS, or whether it somehow filters the project list to ONLY show these. Could you update the comment here? Maybe "List of project codes to filter out and hide from ACAS"?

Reply:
Yeah, I will make this better.